src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -270,6 +271,24 @@ private void triggerChangePullBeforeOverride() {
   }
 
 
+  private void triggerOverwriteExistingFiles() {
+    if (overwriteExistingFilesCheckBox.isSelected()) {
+      requestConfirmation();
+    }
+    onChangeOverwriteExistingFilesCallbacks.forEach(callback -> callback.accept(overwriteExistingFilesCheckBox.isSelected()));
```

Comment:
We should call the callbacks only if the user confirms that she wants to check the overwrite conf.

Reply:
Ok